exchanges/bittrex/bittrex_wrapper.go

```
@@ -524,8 +689,11 @@ func (b *Bittrex) GetDepositAddress(cryptocurrency currency.Code, _ string) (str
 	if err != nil {
 		return "", err
 	}
+	if depositAddr.Status != "PROVISIONED" {
+		return "", nil
```

Can we return an error here?